bin/polyply

```diff
@@ -77,6 +77,10 @@ def main(): # pylint: disable=too-many-locals,too-many-statements
     dna_group = parser_gen_itp.add_argument_group('DNA specifc options')
     dna_group.add_argument('-dsdna', dest='dsdna', action='store_true',
                            help='complement single sequence to dsDNA sequence')
+    protein_termini_group = parser_gen_itp.add_argument_group('Protein termini group')
+    protein_termini_group.add_argument('-ter', dest='ter', action='store_true',
+                                       help='patch termini with correct parameters'
+                                       )
```

Comment:
I'm open to having this not only for the proteins. Perhaps later down the line it can just be a regular argument depending how it shapes up.